src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -59,6 +59,7 @@ private void handleAlert(String id, GtfsRealtime.Alert alert) {
         alertText.alertDescriptionText = deBuffer(alert.getDescriptionText());
         alertText.alertHeaderText = deBuffer(alert.getHeaderText());
         alertText.alertUrl = deBuffer(alert.getUrl());
+        alertText.setId(id);
```

Comment:
should this be `alertText.id = id` to match the existing style?

Reply:
Existing style (previous lines) is about public fields, which are exposed through OTP's API. Newly added `id` field is neither public, nor exposed through OTP API (there is no reason for it to be), so I'd say that it shouldn't.